packages/cli/src/resource/strings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "error": {
+    "NotAllowedMigrationErrorMessage": "Teams toolkit's pre-released version supports new project configuration and is incompatible with previous versions. Please try it by creating a new project or run \"teamsfx upgrade\" to upgrade your project first."
```

Comment:
Teams toolkit -> Teams Toolkit
pre-released -> Pre-Release
remove "Please" and try -> Try

Reply:
Thank you, and resolved.